.github/workflows/ci.yml

```diff
@@ -15,6 +15,10 @@ on:
         required: false
         default: false
 
+defaults:
+  run:
+    shell: bash -el {0}
+
```

Comment:
This change is easy enough to port over to all the other shells but it'll be good to keep an eye on the mambaforge actions repo to see if they come up with a clever way to avoid this.